src/Dapr.Extensions.Configuration/DaprSecretStoreConfigurationSource.cs

```diff
@@ -64,11 +70,11 @@ public IConfigurationProvider Build(IConfigurationBuilder builder)
                     throw new ArgumentException($"{nameof(Metadata)} must be null when {nameof(SecretDescriptors)} is set", nameof(Metadata));
                 }
 
-                return new DaprSecretStoreConfigurationProvider(Store, NormalizeKey, KeyDelimiters, SecretDescriptors, Client);
+                return new DaprSecretStoreConfigurationProvider(Store, NormalizeKey, KeyDelimiters, SecretDescriptors, Client, SidecarWaitDuration ?? TimeSpan.FromSeconds(5));
```

Comment:
minor nitpick: It would be great to have ... 1 place in code where the default timeout lives. It feels wierd when multiple layers of the code all set it.

Reply:
Given that it's spread out over multiple classes, this definitely should be moved to a constant.

Reply:
Made it a constant.